### DISCUSSION TITLE
Tree: Is it possible to not collapse a expanded TreeNode on DragStart?

### DISCUSSION THREAD

**Discussion by HolgerSeithHmh** [ant-design/ant-design#43968] [Q&A]
If you start dragging a expanded TreeNode the dragged TreeNode is collapsed (even if it is not dropped elsewhere).

See example of draggable ANT tree: https://ant.design/components/tree#components-tree-demo-draggable

Is this a bug? (in my opinion yes)
Or is this behavior intentially and I have to control "expandedKeys" and "onExpand" ?

**Comment by HolgerSeith** [comment]
no one ? :(
